lib/datadog/tracing/contrib/rack/middlewares.rb

```diff
@@ -35,22 +35,31 @@ def compute_queue_time(env)
             request_start = Contrib::Rack::QueueTime.get_request_start(env)
             return if request_start.nil?
 
-            frontend_span = Tracing.trace(
-              Ext::SPAN_HTTP_SERVER_QUEUE,
-              span_type: Tracing::Metadata::Ext::HTTP::TYPE_PROXY,
-              start_time: request_start,
-              service: configuration[:web_service_name]
-            )
-
-            # Tag this span as belonging to Rack
-            frontend_span.set_tag(Tracing::Metadata::Ext::TAG_COMPONENT, Ext::TAG_COMPONENT)
-            frontend_span.set_tag(Tracing::Metadata::Ext::TAG_OPERATION, Ext::TAG_OPERATION_HTTP_SERVER_QUEUE)
-            frontend_span.set_tag(Tracing::Metadata::Ext::TAG_KIND, Tracing::Metadata::Ext::SpanKind::TAG_SERVER)
```

Comment:
It's worth noting that these tags aren't arbitrary display, but are conventional and used to drive certain behaviors within Datadog. I think it's a good idea if we preserve `kind`, `component` and similar tags on the queue span, in case Datadog needs these for some queue-specific feature in the UI.

 - `service` reflects the application name of the process that performs the operation. Because "http server request" and "queue" represent time spent in the proxy (not in Rack), I think they should reflect `configuration[:web_service_name]`. This will help make the statistics in the UI clear, too.
 - `component` is supposed to describe which library a span is associated with. The "rack request" would be `rack`, whereas the "http server request" and "queue" would ideally reflect the library responsible (e.g. `nginx`). If we're unable to determine an obviously "correct" value from the request, I think we could attribute it to `http_server` (or something similar) to those latter two.
 
   Long term, Ruby shouldn't produce these "http server request" and "queue" spans at all; the proxy server should be instrumented and determine this name for itself.
 - `kind` reflects the role of the thing performing the action (e.g. `server` or `client`). All these spans should be tagged as `server` I think.